### PR TITLE
Add Doc for ContainerInsights EKS Prometheus

### PR DESCRIPTION
**Pull request by bjrara**
Add public doc for sContainerInsights EKS Prometheus. This doc is a part of the next release for AWS OpenTelemetry Collector.

### CODE REVIEW

**Comment by pingleig** [comment]
Preview https://trusting-lewin-3415ee.netlify.app/docs/getting-started/container-insights/eks-prometheus

btw: @bjrara you seems forgot the screenshots for viewing dashboards ...

**Review by mxiamxia — APPROVED** [review]
Thanks

**Comment by bjrara** [comment]
> Preview https://trusting-lewin-3415ee.netlify.app/docs/getting-started/container-insights/eks-prometheus
> 
> btw: @bjrara you seems forgot the screenshots for viewing dashboards ... and the breadcrumb for prometheus is broken because I removed the index page for prometheu, the latest should be
> 
> ```
> <p>
>     GettingStarted / <Link to="/docs/getting-started/container-insights">Container Insights</Link> / <Link to="/docs/getting-started/container-insights#collecting-prometheus-metrics">Container Insights Prometheus</Link>
> </p>
> ```

Thanks for the suggestions. Done.

**Review by alolita — COMMENTED** [review]
Fixed spelling and usage. Do not use AWS OTEL Collector. Please use ADOT Collector instead.

**Review by alolita — CHANGES_REQUESTED** [review]
Please correct all usage of the following:
* AWS OTel Collector to ADOT Collector
* Container Insight Prometheus - this cannot be used - trademark violation ; you can use Container Insights support for Prometheus metric types
* collector to Collector

File: src/docs/getting-started/container-insights/eks-prometheus.mdx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import { Link } from "gatsby"
+import dashboardImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-dashboard.png"
+import logsImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-logs.png"
+import metricsImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-metrics.png"
+
+<p>
+    GettingStarted / <Link to="/docs/getting-started/container-insights">Container Insights</Link> / <Link to="/docs/getting-started/container-insights#collecting-prometheus-metrics">Container Insights Prometheus</Link>
+</p>
+
+This section explains how to set up Prometheus monitoring with the ADOT Collector in a cluster running Amazon EKS or Kubernetes. With [default configurations](https://github.com/aws-observability/aws-otel-collector/tree/main/config/eks/prometheus), the AWS OTel Collector automatically scrapes and imports metrics for the following workloads running in a cluster.
```
Comment (mxiamxia) [inline src/docs/getting-started/container-insights/eks-prometheus.mdx:15]:
```suggestion
This section explains how to set up Prometheus monitoring with the ADOT Collector in a cluster running Amazon EKS or Kubernetes. With [default configurations](https://github.com/aws-observability/aws-otel-collector/tree/main/config/eks/prometheus), the ADOT Collector automatically scrapes and imports metrics for the following workloads running in a cluster.
```

File: src/docs/getting-started/container-insights/eks-prometheus.mdx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+AmazonEC2ContainerRegistryReadOnly
+CloudWatchAgentServerPolicy
+AmazonSSMManagedInstanceCore
+AWSXrayWriteOnlyAccess
+AWSXRayDaemonWriteAccess
+AmazonS3ReadOnlyAccess
+AmazonPrometheusFullAccess
+```
+
+## Deploy the AWS OTel Collector with default configuration
```
Comment (mxiamxia) [inline src/docs/getting-started/container-insights/eks-prometheus.mdx:39]:
```suggestion
## Deploy the ADOT Collector with default configuration
```

File: src/docs/getting-started/container-insights/eks-prometheus.mdx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+      Prometheus metrics before awsemf exporter sending metric logs to CloudWatch.
+
+1. Create ConfigMap `otel-config` in the `aws-otel-eks` namespace with your customized configuration so that it can be consumed by aws-otel-collector Pods.
+  
+  Be sure to rename `{{filename}}` with the actual filename.
+  ```bash
+  kubectl create configmap otel-config --from-file={{filename}} -naws-otel-eks
+  ```
+
+### Run aws-otel-collector
```
Comment (mxiamxia) [inline src/docs/getting-started/container-insights/eks-prometheus.mdx:101]:
```suggestion
### Run ADOT Collector
```

File: src/docs/getting-started/container-insights/eks-prometheus.mdx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+path: '/docs/getting-started/container-insights/eks-prometheus'
+---
+
+import { Link } from "gatsby"
+import dashboardImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-dashboard.png"
+import logsImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-logs.png"
+import metricsImg from "assets/img/docs/gettingStarted/container-insights/eks-prometheus-metrics.png"
+
+<p>
+    GettingStarted / <Link to="/docs/getting-started/container-insights">Container Insights</Link> / <Link to="/docs/getting-started/container-insights#collecting-prometheus-metrics">Container Insights Prometheus</Link>
```
Comment (mxiamxia) [inline src/docs/getting-started/container-insights/eks-prometheus.mdx:12]:
```suggestion
    GettingStarted / <Link to="/docs/getting-started/container-insights">Container Insights</Link> / <Link to="/docs/getting-started/container-insights#collecting-prometheus-metrics">Container Insights for Prometheus Support</Link>
```

**Review by alolita — APPROVED** [review]

**Pull request merged**